json_object.c

```diff
@@ -474,15 +474,28 @@ static int json_object_object_to_json_string(struct json_object *jso, struct pri
 		if (flags & JSON_C_TO_STRING_SPACED && !(flags & JSON_C_TO_STRING_PRETTY))
 			printbuf_strappend(pb, " ");
 		indent(pb, level + 1, flags);
+		if (flags & JSON_C_TO_STRING_COLOR)
+			printbuf_strappend(pb,"\033[0;34m");
```

Comment:
Instead of having rather opaque escape sequences scattered around it would be better to add some defines and use them instead.  e.g. perhaps:
```
#define ANSI_COLOR_RESET "\e[0m"
#define ANSI_COLOR_FG_GREEN "\e[0;32m"
#define ANSI_COLOR_FG_BLUE "\e[0;34m"
#define ANSI_COLOR_FG_MAGENTA "\e[0;35m"
...
   printbuf_strappend(pb, ANSI_COLOR_FG_BLUE);
...
```
etc...

I'm not sure if there are commonly used names for these; if they exist, it would be a good idea to find and use those names.